scripts/dmsdk/gen_cpp.py

```diff
@@ -219,6 +287,10 @@ def gen_cpp_header(basepath, c_header_path, out_path, info, ast, state, includes
     l('')
     l('#endif // #define DMSDK_' + basename.upper())
 
+    # Workaround, until we have proper multi language support.
+    # We attach the C documentation at the end of this file.
+    add_c_docs(docs)
```

Comment:
The current "big hack":
I adde the entire documentation of the C file into the documentation of the generated C++ file, as it's currently the one being picked up by our documentation build system.

True, multi language support comes later.